polkadot/zombienet_tests/smoke/0004-coretime-smoke-test.zndsl

```diff
@@ -9,7 +9,7 @@ coretime-collator: is up
 alice: js-script ./0004-configure-relay.js with "" return is 0 within 600 secs
 
 # Coretime chain should be producing blocks when the extrinsic is sent
-alice: parachain 1005 block height is at least 10 within 120 seconds
+alice: parachain 1005 block height is at least 10 within 180 seconds
```

Comment:
is this so high because we need to wait two sessions?

Reply:
Afaik it's one session (10 blocks which is around a minute). On top of this we need some time to get to 10 blocks worst case (for my local run it was around 7 block) which is another 40 secs. Or 100 secs total.

Initially I thought 120secs should be enough to finish this but I noticed a job failed at this step so I decide to put one more minute to be on the safe side.

Reply:
maybe let's leave 8 seconds for each block (which is what we do in other tests AFAIK)

Reply:
Just to make sure we are on the same page. As soon as `./0004-configure-relay.js` executes the coretime chain stalls. It unstalls after one complete session. 

So we have:
- a few blocks to let the old session finish (let's say 5?)
- 10 blocks for a new session
- 7 blocks to reach the threshold after the change kicks in 

`(5+10+7)*8=176 secs` for a timeout which by pure luck is around my estimate.

Maybe it's safe to decrease the threshold to 6 with a timeout of 150 secs?

Reply:
> So we have:

> a few blocks to let the old session finish (let's say 5?)
> 10 blocks for a new session

Ok so indeed two sessions need to pass. It's ok then

Reply:
Sorry, I've misunderstood you initially. I thought you mean two full sessions.